contk/dataloader/language_generation.py

```diff
@@ -201,15 +217,15 @@ def trim_index(self, index):
 				* ignore `<pad>` s at the end of the sentence.
 
 		Arguments:
-				index (list): a list of int
+			index (list): a list of int
 
 		Examples:
-				>>> dataloader.index_to_sen(
-				...		[])
-				>>>
+			vocab_list = ["<pad>", "<unk>", "<go>", "<eos>", "I", "have",\
+						  "been", "to", "Sichuan"]
+			>>> dataloader.trim_index(
+			...		[2, 4, 5, 6, 7, 8, 3, 0, 0])
+			>>> [2, 4, 5, 6, 7, 8]
 
```

Comment:
Add a more complex example to show the corner case:.

`           >>> dataloader.trim_index(
			...		[2, 4, 5, 6, 7, 8, 0, 0, 3, 4, 5, 3, 0, 0])
			>>> [2, 4, 5, 6, 7, 8]`

Add some comments for readability （turn list of int to words)